### PR TITLE
flatbuffers:AddedScripts

### PR DESCRIPTION
**Pull request by ankitpar**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [ ] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

File: f/flatbuffers/flatbuffers_ubi8.5.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+# -----------------------------------------------------------------------------
+#
+# Package       : flatbuffers
+# Version       : v1.12.0
+# Source repo   : https://github.com/google/flatbuffers.git
+# Tested on     : UBI 8.5
+# Language      : C++
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation {maintainer}
```
Comment (seth-priya) [inline f/flatbuffers/flatbuffers_ubi8.5.sh:11]: Please put you name and email address in maintainer

File: f/flatbuffers/flatbuffers_ubi8.5.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=flatbuffers
+PACKAGE_VERSION=v1.12.0
+PACKAGE_URL=https://github.com/google/flatbuffers.git
+
+yum install -y git wget gcc-c++ cmake
+yum -y update --allowerasing --skip-broken --nobest
```
Comment (seth-priya) [inline f/flatbuffers/flatbuffers_ubi8.5.sh:26]: why is this needed?

**Review by seth-priya — CHANGES_REQUESTED** [review]

File: f/flatbuffers/flatbuffers_ubi8.5.sh (hunk 1 above)
Comment (ankitpar) [reply f/flatbuffers/flatbuffers_ubi8.5.sh]: Fixed

File: f/flatbuffers/flatbuffers_ubi8.5.sh (hunk 2 above)
Comment (ankitpar) [reply f/flatbuffers/flatbuffers_ubi8.5.sh:26]: its to resolve version issue with already installed packages.

Comment (seth-priya) [reply f/flatbuffers/flatbuffers_ubi8.5.sh]: removed this line and the build / travis check passed

**Review by seth-priya — APPROVED** [review]
